ui-tests/tests/widget.test.ts

```diff
@@ -79,6 +79,99 @@ test('test create new connection', async ({ page }) => {
 
 });
 
+
+const labels_in_memory = ['DuckDB', 'SQLite']
+
+
+test('test irrelevant fields in memory db', async ({ page }) => {
+    await displayWidget(page);
+    await page.locator('#createNewConnection').click();
+    for (const label of labels_in_memory) {
+    await page.locator('#selectConnection').selectOption({ label: label });
+    const username = page.locator('#username');
+    expect(await username.count()).toBe(0);
+    const password = page.locator('#password');
+    expect(await password.count()).toBe(0);
+    const host = page.locator('#host');
+    expect(await host.count()).toBe(0);
```

Comment:
some indentation and spacing between `expect` lines would make this more readable

Reply:
[fixed](https://github.com/ploomber/jupysql-plugin/pull/73/commits/769403be5bc29b39d4c6d84c8a81607db59b7110)